authz/providers/azure/rbac/rbac.go

```diff
@@ -0,0 +1,198 @@
+/*
+Copyright The Guard Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package rbac
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path"
+	"strconv"
+	"time"
+
+	"github.com/appscode/guard/auth/providers/azure/graph"
+	"github.com/golang/glog"
+	"github.com/moul/http2curl"
+	"github.com/pkg/errors"
+	authzv1 "k8s.io/api/authorization/v1"
+)
+
+var (
+	MANAGED_CLUSTER   = "Microsoft.ContainerService/managedClusters"
```

Comment:
it's atypical to use all cap in golang. if it's private, use camelCasing and start with lower case